packages/core/src/browser/tree/tree-widget.tsx

```diff
@@ -844,7 +844,7 @@ export class TreeWidget extends ReactWidget implements StatefulWidget {
     }
 
     protected getPaddingLeft(node: TreeNode, props: NodeProps): number {
-        return props.depth * this.props.leftPadding + (this.needsExpansionTogglePadding(node) ? this.props.expansionTogglePadding : 0);
```

Comment:
I believe this change will not work once we have the preference to control the behavior of the indent guide.

Reply:
I’m pretty sure it breaks some trees or themes. If we change something in general tree, we have to retest all possible combinations of trees and file icon themes, i.e. default, none, seti (font based), minimal (image based(.